draft-ietf-core-cf-reg-update.md

```diff
@@ -38,53 +38,56 @@ normative:
   RFC7120: iana-early
   RFC7252: coap
   RFC9110: http-sema
+  RFC9193: senml-cf
```

Comment:
I know nobody sees this, but s/cf/ct/

Reply:
done in [182822b](https://github.com/core-wg/cf-reg-update/pull/40/commits/182822b1aed08eb3453d1a76f911649a97b5b5c7)